package/yast2.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May  9 13:41:51 CEST 2016 - snwint@suse.de
+
+- remove invalid_hostname check (fate #319639)
```

Comment:
I think the guardian robots do not like the space between "fate" and "#".
You can run the check locally with `rake check:changelog`